### DISCUSSION TITLE
Wiki Fandom support?

### DISCUSSION THREAD

**Discussion by AtagoOkino** [Frederisk/Wikitext-Extension-Gadget#7] [Q&A] [answered]
Hi, your extension looks very very good. 
But I'm a mainstay at Wikifandom. I wonder if this extension can support Wiki Fandom since they all have Wiki bases. If so, how can I install it?

**Comment by Frederisk** [comment, marked as answer]
My Preferences > Appearance > Skin preferences > Custom JavaScript & Enable personal JavaScript

**Reply by AtagoOkino** [reply]
Thank you very much, it works so great!
![image](https://github.com/Frederisk/Wikitext-Extension-Gadget/assets/90787570/f51761b5-57d6-43ee-8b7d-521338492d54)
